README.md

```diff
@@ -257,6 +257,23 @@ Settings:
 
 Clone repo or pull latest changes to specified repo
 
+#### git.blame-copy
+
+Command for creating copies of a file with git blame history saving.
+
+Original script written in bash:
+https://dev.to/deckstar/how-to-git-copy-copying-files-while-keeping-git-history-1c9j
+
+Usage:
+```shell
+  inv git.blame-copy <path to original file> <path to copy>,<path to copy>...
+```
+
+Settings:
+
+* `copy_commit_template` template for commits created during command workflow
+* `copy_init_message_template` template for init message printed at command start
```

Comment:
Maybe also add examples of templates or at least describe all possible "template variables"